README.md

```diff
@@ -28,11 +28,12 @@ Setup for developers
   shell variable every time you activate the virtualenv, edit `venv/bin/activate`
   and add to the bottom the export statement.
 7. Run `python systers_portal/manage.py migrate`.
-8. Run `python systers_portal/manage.py runserver` to start the development server. When in testing
+8. Run `python systers_portal/manage.py createsuperuser` to create a superuser for the admin pannel. Fill in the details asked.
```

Comment:
This is a good message. Also try to keep lines under 80 characters as much as possible. You can start "Fill in the detailed asked." sentence from a new line and don't forget to indent it with 2 space, such that markdown parser will treat it as a single bullet point in one line.